From: Andre Heider <a.heider@gmail.com>
Date: Wed, 17 Mar 2021 09:54:47 +0100
Subject: meson: update to 0.57.1

v0.57 is the first version that requires at least python 3.6.
Since commit 7379f8bd on the main repository we can now rely on this
prerequisite.

Switch to AUTORELEASE while at it.

Signed-off-by: Andre Heider <a.heider@gmail.com>

diff --git a/devel/meson/Makefile b/devel/meson/Makefile
index b90f511afe846a438ceef8551d6c80bb605d4517..9142b1f8793a253888089559bf95b5dd2a113e34 100644
--- a/devel/meson/Makefile
+++ b/devel/meson/Makefile
@@ -1,11 +1,11 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=meson
-PKG_VERSION:=0.56.2
-PKG_RELEASE:=1
+PKG_VERSION:=0.57.1
+PKG_RELEASE:=$(AUTORELEASE)
 
 PYPI_NAME:=meson
-PKG_HASH:=3cb8bdb91383f7f8da642f916e4c44066a29262caa499341e2880f010edb87f4
+PKG_HASH:=72e1c782ba9bda204f4a1ed57f98d027d7b6eb9414c723eebbd6ec7f1955c8a6
 
 PKG_MAINTAINER:=Andre Heider <a.heider@gmail.com>
 PKG_LICENSE:=Apache-2.0
